Gemfile

```diff
@@ -53,7 +53,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'blacklight', "~> 6.7"
-gem 'trln_argon', git: 'https://github.com/trln/trln_argon'
+gem 'trln_argon', git: 'https://github.com/trln/trln_argon', tag: 'v0.5.16'
```

Comment:
Sean, is there a reason this is limited to a single tagged release?

Reply:
@corylown Good question -- would it be better to `bundle update trln_argon` then commit files to the `.lock` file when updates from the engine are needed?

Reply:
That's what I'd do (or `bundle update --source trln_argon`), unless there's some reason we don't want whatever the latest version of Argon is.

Reply:
@corylown OK thanks -- will revise that shortly